src/utils.py

```
@@ -216,19 +216,14 @@ def _check_version(output: str) -> None:
 
 
 def check_java() -> None:
-    """The function `check_java` checks if Java version 17 or higher is installed.
-
-    Returns
-    -------
-        The function `check_java` does not return any value.
-    """
+    """Check that the installed Java version meets the minimum requirement."""
     try:
         jd = subprocess.check_output(["java", "-version"], stderr=subprocess.STDOUT).decode("utf-8")
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)